packages/client/src/base/feedback/feedback-action-dispatcher.ts

```diff
@@ -57,6 +71,16 @@ export interface IFeedbackActionDispatcher {
      * Retrieve all `actions` sent out by currently registered `feedbackEmitter`.
      */
     getRegisteredFeedback(): Action[];
+
+    /**
+     * Retrieves all commands based on the registered feedback actions, ordered by their priority (highest priority first).
+     */
+    getFeedbackCommands(): Command[];
+
+    /**
+     * Applies all current feedback commands to the given command execution context.
+     */
+    applyFeedbackCommands(context: CommandExecutionContext): Promise<void>;
```

Comment:
This introduces a (minor) API break for potential `IFeedbackActionDispatcher` implementations.
At the very least we should mention it in the changelog/migration guide.

Reply:
I could mark the method as optional, that would allow for backwards compatibility, would you prefer that?

Reply:
No, I don't think making it optional is the right way to go here.
I think it this case it is fine that we introduce a soft API break. We should just mention it in the changelog.
Also it's probably very unlikely that somebody uses a complete custom `IFeedbackActionDisptacher` implementation.
If adopters have custom code they probably extended the `FeedbackActionDispatcher` base class anyways. 
So in practice this break should have little to no impact.